__tests__/sagas/wallet.test.ts

```
@@ -0,0 +1,186 @@
+  startWallet,
+  onResetWallet,
+  WALLET_STATUS,
+  isWalletServiceEnabled,
+  isPushNotificationEnabled,
+} from '../../src/sagas/wallet';
+import { STORE } from '../../src/store';
+import { PRE_SETTINGS_MAINNET } from '../../src/constants';
+import { getNetworkSettings } from '../../src/sagas/helpers';
+
+// Suppress noisy wallet-lib logs during tests
+jest.mock('../../src/logger', () => ({
```

ESLint

[error] 'jest' is not defined.

(no-undef)